### ISSUE TITLE
Autocomplete/matching list of functions for python scripting interface

### ISSUE THREAD

**Issue by olibclarke** [pemsley/coot#83]
Hi,

In 0.9x, when typing in the python scripting interface, a list of functions matching the entered strings appeared. It would be useful to have this feature also in 1.x (maybe enhanced to provide info on usage/arguments...?) as it makes on-the-fly scripting significantly easier.

Cheers
Oli

**Comment by olibclarke** [comment]
Ah I see, it is there! They all just start with `coot.whatever` now - that is a change from the old version, got it

**Comment by olibclarke** [comment]
Hmm I spoke too soon... it worked the first time I tried it, but then subsequent times in the same session the panel comes up, but remains blank, even after `import coot`:

<img width="1265" alt="image" src="https://github.com/pemsley/coot/assets/19766818/0f9ef683-74f0-4d3f-ac78-94e271928b76">

**Comment by olibclarke** [comment]
Another thing on this one - when the list of functions does appear, it is not scrollable (although it looks like it ought to be - there is a scrollbar), and items in the list cannot be clicked (though they can be selected using the arrow keys)

**Comment by olibclarke** [comment]
The scripting text entry box also overlaps with the lefthand panels (e.g. the overlaps panel)

<img width="1016" alt="image" src="https://github.com/pemsley/coot/assets/19766818/4f8f4d90-da24-4697-89ed-95f8a44abefb">

**Comment by pemsley** [comment]
"Escape" to dismiss the Python interface.

**Comment by pemsley** [comment]
Focus is different now in GTK4. I don't know how it works. That's why, for the moment, there is a "Grab Focus" button which puts the focus of the keyboard back into the graphics widget.

**Comment by olibclarke** [comment]
Ah, ok. Maybe for the meantime it would be better if the scripting interface was in a separate window like in GTK3 or GTK2? That way it is more clear what has focus...?

**Comment by pemsley** [comment]
FWIW, scrolling works for me with the arrow keys when the completion dialog is show.

To recall a previous command use Shift-Ctrl-UpArrow.

**Comment by olibclarke** [comment]
Yes, arrow keys work, but clicking the items in the list does not select them.

**Comment by pemsley** [comment]
Sounds like the Quartz -> GTK events bug again.
